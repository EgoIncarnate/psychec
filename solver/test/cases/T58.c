void f() {
  T a;
  a = (float)e;
}

