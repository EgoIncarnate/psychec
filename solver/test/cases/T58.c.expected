typedef float T;
uintptr_t e;
